web/styles/app_components.css

```diff
@@ -192,7 +192,8 @@ input::placeholder {
 
         &:hover,
         &:focus {
-            border-color: hsl(0deg 0% 60%);
+            border-color: hsl(0deg 0% 60%) !important;
```

Comment:
Why do we need `!important` here? Is it because of a conflict with dark theme styles? This seems like a case where the right fix is to define color variables properly instead of an override in `dark-theme.css`...

Reply:
@timabbott I defined color variables in zulip.css.